READMEquay.md

```diff
@@ -0,0 +1,90 @@
+# Building with quay.io
+
+## quay.io repositories
+
+quay.io is a convenient place to build and store you development 
+docker images.  Set up an account for yourself and then set up
+an automatic build of your ovn-kubernetes fork. The build is
+triggered  whenever you push changes to your fork. The build is
+on the master branch.  The result is
+```
+quay.io/repository/<your-account>/ovn-kube:latest
+```
+You can use the image in your daemonsets for development.
+
+Dockerfile.centos builds the binaries and then includes them in the 
+image.
+
+
+## setup docker pull access on each host in the cluster
+Edit **/etc/containers/registries.conf**
+
+```
+registries:
+  - brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888
```

Comment:
I'd take this registry out since it's an internal one that nobody else can use.

Reply:
Fixed